src/content/en/2021/caching.md

```diff
@@ -17,4 +19,303 @@ featured_stat_3: TODO
+
+The median HTML resource on mobile has a TTL of 14 days
+
+## Cookie Cache adoption
+
+35.63% of Cacheable Desktop pages with set-cookie
+Mobile pages with set-cookie: 35.99%
+
+Another area of opportunity we discovered was with cacheable pages that are not using `set-cookie`. About 35% of Desktop and Mobile pages use `set-cookie` while over 64% of them do not.
+
+Large area of opportunity for pages that are actually cacheable, but are just missing this directive?
+
+Fun fact `test_cookie` is the most popular desktop and mobile cookie name at 4% of all cookies.
+
+{{ figure_markup(
+  caption="TODO",
+  content="4%",
+  classes="big-number",
+  sheets_gid="455647224",
+  sql_file="todo.sql"
+) }}
+
+The top cookies attributes used on Desktop and Mobile are SameSite and Secure. SameSite is on 53% of cookies set in Desktop responses and a whopping 68% on Mobile responses.
+
+Why is it a big deal that SameSite is at 68% on mobile?
+
+## What type of content are we caching?
+
+Talk about what kind of resources are we caching, (resource age groups: party, type).
+
+The median age of HTML resources cached for mobile is 9 weeks, while for text the median age is 1220 weeks (23.4 years).
+
+Talk about, total number of requests with comparable Last-Modified and expiration times, not necessarily all requests.
+
+Talk about 54% of mobile resources are older than their TTL.
+
+{{ figure_markup(
+  caption="TODO",
+  content="54%",
+  classes="big-number",
+  sheets_gid="768623684",
+  sql_file="todo.sql"
+) }}
+
+Total number of requests with comparable Last-Modified and expiration times, not necessarily all requests.
+
+Talk about, 60% of 1P mobile resources are older than their TTL, which is more common than 3P resources (45%).
+
+Explain why this is significant- 60% of 1st Party mobile resources are older than their TTL- is this because 1P are more important or crucial? Because 3P are being deprecated?
+
+## How do cache TTLs compare to resource age?
+
+The median difference between resource TTL and resource age is -6 days, meaning the age is 6 days older than the TTL.
+
+## Identifying caching opportunities
+
+Can we get a graph of the % of cacheable strategies vs. non-cacheable strategies for Desktop and Mobile, based on data from 'Non-cacheable strategies'?
+
+Shows the majority of resources are highly cacheable
+
+{{ figure_markup(
+  image="caching-by-resource-type.png",
+  caption="The percent of requests that use caching strategies by resource type.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=2036781114&format=interactive",
+  height=436,
+  sheets_gid="1202769738",
+  sql_file="non_cacheable_by_resource_type.sql"
+) }}
+
+### Lighthouse TTL score
+Talk about 12% of mobile pages score 0.9 or higher on the long Time To Live (TTL) audit
+
```

Comment:
Based on the Lighthouse caching TTL score, we have seen an improvement in pages ranked with a perfect 100 score increase from 3.3 in 2020 to 4.4 in 2021. 
The score reflects whether the pages can benefit from additional caching policy improvements. Even though we are excited to see 31% of pages scoring above the 50 percentile score, we still see a large potential in 52% of pages that have a below-25 percentile score. This makes us consider that even though web pages have some level of caching, the way the policies are used is outdated and not optimized to the latest state of their products.